src/lang/en-US.json

```diff
@@ -184,6 +184,12 @@
   "filter_last_month": "Last month",
   "filter_last_3_months": "Last 3 months",
   "filter_last_6_months": "Last 6 months",
+  "filter_with_games_played": "At least this many games played",
```

Comment:
suggestion from earlier about not having an individual string for each option still stands

Reply:
actually you can reuse the string from the Explorer:
"explorer_having"